### PR TITLE
Support cross-building CoreCLR for arm64 Android

### PR DESCRIPTION
**Pull request by qmfrederik**
This PR patches the CMake files so that they support cross-building CoreCLR for arm64 Android.

An overview of the changes:
* Fix the path to `objcopy`
* Don't run `verify-so.sh` when cross-building for Android
* Don't link with gcc_s, m and pthread on Android (I confirm there is no gcc_s); but do link with gnustl_shared, android-support, android-glob and intl
* Link with `unwind-aarch64` on arm64

### CODE REVIEW

**Comment by qmfrederik** [comment]
@janvorli One more PR for Android.

**Comment by qmfrederik** [comment]
@janvorli The `exception_handling.pal_sxs.test1` tests seems to be failing on Ubuntu, which I find weird - see https://ci.dot.net/job/dotnet_coreclr/job/master/job/checked_ubuntu_prtest/2277/testReport/

Might this be unrelated?

**Comment by qmfrederik** [comment]
@dotnet-bot help

**Comment by qmfrederik** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test

**Comment by qmfrederik** [comment]
@dotnet-bot test Windows_NT x64 Debug Build and Test

File: src/pal/src/CMakeLists.txt [hunk 1]
```diff
@@ -281,31 +281,73 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
...
 
-  find_library(UNWIND NAMES unwind)
+  if(NOT CLR_CMAKE_PLATFORM_ANDROID)
+    find_library(UNWIND NAMES unwind)
+
+    if(UNWIND STREQUAL UNWIND-NOTFOUND)
+      message(FATAL_ERROR "Cannot find libunwind. Try installing libunwind8-dev and libunwind8.")
+    endif(UNWIND STREQUAL UNWIND-NOTFOUND)
+
+    target_link_libraries(coreclrpal ${UNWIND})
```
Comment (janvorli) [inline src/pal/src/CMakeLists.txt]: This is the reason for the test failure. The Unwind libraries must be specified after the gcc_s - that's what I have mentioned to you some time ago.

Comment (qmfrederik) [inline src/pal/src/CMakeLists.txt]: You are right. It should be fixed now.

**Pull request merged**